rotkehlchen/inquirer.py

```diff
@@ -507,36 +512,43 @@ def _find_price(
             ignore_cache: bool = False,
             skip_onchain: bool = False,
             coming_from_latest_price: bool = False,
-    ) -> Tuple[Price, CurrentPriceOracle]:
-        """Returns the current price of 'from_asset' in 'to_asset' valuation.
+            match_main_currency: bool = False,
+    ) -> Tuple[Price, CurrentPriceOracle, bool]:
+        """Returns:
+        1. The current price of 'from_asset' in 'to_asset' valuation.
+        2. Oracle that was used to get the price.
+        3. Flag which is True if the returned price is in the user's main currency, not `to_asset`.
```

Comment:
Again here the question why not to asset? What if to asset == user's main currency?